src/redisgears.h

```diff
@@ -541,13 +541,13 @@ void MODULE_API_FUNC(RedisGears_AddConfigHooks)(BeforeConfigSet before, AfterCon
 #define REDISGEARS_MODULE_INIT_FUNCTION(ctx, name) \
         RedisGears_ ## name = RedisModule_GetSharedAPI(ctx, "RedisGears_" #name);\
         if(!RedisGears_ ## name){\
-            RedisModule_Log(ctx, "warning", "could not initialize RedisGears_" #name "\r\n");\
+            RedisModule_Log(ctx, "warning", "could not initialize RedisGears_" #name);\
```

Comment:
@filipecosta90 this file comes from geras repo, i don't have an issue with updating it but every time we gonna sync with gears' version we will override this changes.

Reply:
agree. removed this change